constrictor.go

```diff
@@ -1,45 +1,36 @@
 package rules
 
+var constrictorRulesetStages = []string{
+	StageMovementStandard,
+	StageStarvationStandard,
+	StageHazardDamageStandard,
+	StageFeedSnakesStandard,
+	StageSpawnFoodStandard,
+	StageEliminationStandard,
+	StageSpawnFoodNoFood,
```

Comment:
Just noticed this - We have both `spawn_food.standard` and `spawn_food.no_food` stages - technically how it worked before, but probably redundant now?

Now I'm wondering if constrictor could just be implemented as this list without either stage?

Reply:
haha yeah - that is a bit silly!

Reply:
oh, I left `spawn_food.no_food`, but I'm seeing now that you mentioned we should remove it

Reply:
... although part of me feels safer with it still in there, just in case...